vendor/vendor.json

```diff
@@ -814,134 +814,134 @@
+			"revisionTime": "2018-02-13T04:10:52Z"
 		},
 		{
 			"checksumSHA1": "zLg5y/6xx/yr8swoqdyxISyQfoU=",
 			"comment": "v0.15.0",
 			"path": "github.com/vmware/govmomi/vim25",
-			"revision": "d68562200280ff2ad2110719b06496c9e9526118",
-			"revisionTime": "2017-10-13T00:55:03Z"
+			"revision": "bb8ff8d53171157123ae812088fd06223d8d90a3",
+			"revisionTime": "2018-02-13T04:10:52Z"
 		},
 		{
 			"checksumSHA1": "xLvppq0NUD6Hv2GIx1gh75xUzyM=",
 			"comment": "v0.15.0",
 			"path": "github.com/vmware/govmomi/vim25/debug",
-			"revision": "d68562200280ff2ad2110719b06496c9e9526118",
-			"revisionTime": "2017-10-13T00:55:03Z"
+			"revision": "bb8ff8d53171157123ae812088fd06223d8d90a3",
+			"revisionTime": "2018-02-13T04:10:52Z"
 		},
 		{
 			"checksumSHA1": "0bHwl55AeUPZ4Jom+K3BHlxVlZA=",
 			"comment": "v0.15.0",
 			"path": "github.com/vmware/govmomi/vim25/methods",
-			"revision": "d68562200280ff2ad2110719b06496c9e9526118",
-			"revisionTime": "2017-10-13T00:55:03Z"
+			"revision": "bb8ff8d53171157123ae812088fd06223d8d90a3",
+			"revisionTime": "2018-02-13T04:10:52Z"
 		},
 		{
 			"checksumSHA1": "kiVMrAiW8EBFQvhDMHdE9SFe3N4=",
 			"comment": "v0.15.0",
 			"path": "github.com/vmware/govmomi/vim25/mo",
-			"revision": "d68562200280ff2ad2110719b06496c9e9526118",
-			"revisionTime": "2017-10-13T00:55:03Z"
+			"revision": "bb8ff8d53171157123ae812088fd06223d8d90a3",
+			"revisionTime": "2018-02-13T04:10:52Z"
 		},
 		{
-			"checksumSHA1": "tN9fHudMKhR3LjGw/V0fM4xXUJw=",
+			"checksumSHA1": "f63BQDwTTtAxMbmVJGmk5K0gWe8=",
 			"comment": "v0.15.0",
 			"path": "github.com/vmware/govmomi/vim25/progress",
-			"revision": "d68562200280ff2ad2110719b06496c9e9526118",
-			"revisionTime": "2017-10-13T00:55:03Z"
+			"revision": "bb8ff8d53171157123ae812088fd06223d8d90a3",
+			"revisionTime": "2018-02-13T04:10:52Z"
 		},
 		{
-			"checksumSHA1": "PSt0tq59P0fFSKvTw/Pn4Ml6u+U=",
+			"checksumSHA1": "dGMLdOsa5PuH4IkB81NphxRiFgA=",
 			"comment": "v0.15.0",
 			"path": "github.com/vmware/govmomi/vim25/soap",
-			"revision": "d68562200280ff2ad2110719b06496c9e9526118",
-			"revisionTime": "2017-10-13T00:55:03Z"
+			"revision": "bb8ff8d53171157123ae812088fd06223d8d90a3",
+			"revisionTime": "2018-02-13T04:10:52Z"
 		},
 		{
-			"checksumSHA1": "SPTCH8wQxDD8AHvNVUgBV35SBZY=",
+			"checksumSHA1": "YcDMYKmKKztKDr197u+bMT8o5tU=",
 			"comment": "v0.15.0",
 			"path": "github.com/vmware/govmomi/vim25/types",
-			"revision": "d68562200280ff2ad2110719b06496c9e9526118",
-			"revisionTime": "2017-10-13T00:55:03Z"
+			"revision": "bb8ff8d53171157123ae812088fd06223d8d90a3",
+			"revisionTime": "2018-02-13T04:10:52Z"
 		},
 		{
 			"checksumSHA1": "6XLRzLEvncx5ORtvBjp41KfRTjc=",
 			"comment": "v0.15.0",
 			"path": "github.com/vmware/govmomi/vim25/xml",
-			"revision": "d68562200280ff2ad2110719b06496c9e9526118",
-			"revisionTime": "2017-10-13T00:55:03Z"
+			"revision": "bb8ff8d53171157123ae812088fd06223d8d90a3",
+			"revisionTime": "2018-02-13T04:10:52Z"
```

Comment:
should we not be pinning to the tag of ` 0.16.0`?

Reply:
Versions in govmomi generally are only useful for govc. The SDK is unversioned. As an example, `0.16.0` was actually released back in November.